packer/plugin/discover.go

```diff
@@ -236,7 +236,11 @@ func (c *Config) discoverMultiPlugin(pluginName, pluginPath string) error {
 
 	for _, builderName := range desc.Builders {
 		builderName := builderName // copy to avoid pointer overwrite issue
-		c.builders[pluginPrefix+builderName] = func() (packersdk.Builder, error) {
+		key := pluginPrefix + builderName
+		if builderName == "default" {
```

Comment:
Do you think using an underscore here `_` could be a good idea ? In case someone wants to call their plugin `default` ?

Reply:
```suggestion
		if builderName == "_" {
```